airqo_device_monitor/constants.py

```diff
@@ -0,0 +1,11 @@
+###### Thingspeak API ######
+
+# feeds
+DEFAULT_THINGSPEAK_FEEDS_INTERVAL_DAYS = 7
+THINGSPEAK_FEEDS_LIST_MAX_NUM_RESULTS = 8000
+THINGSPEAK_FEEDS_LIST_URL = 'http://thingspeak.com/channels/{}'
+
+# channels
+THINGSPEAK_CHANNELS_LIST_URL_TEMPLATE = 'https://api.thingspeak.com/users/{}/channels.json'
```

Comment:
is this necessary, or just for the later constant?

Reply:
just for the late constant, but i separated it out in case we wanted to use that username elsewhere then it would be in one place